### DISCUSSION TITLE
Varias consultas TP1

### DISCUSSION THREAD

**Discussion by Clirussa** [INGCOM-UNRN-P1/cursada-2023#69] [Preguntas y respuestas] [answered]
Hola, para el punto 1, cuando dice "SOLDOUT" solo se refiere a imprimir en pantalla el Hola mundo, no? o también tiene que ver con alguna forma de codificar, o de hacer el ejercicio? 

En algunos puntos, aparecen por ejemplo:

float centigrados_a_fahrenheit(float centigrados); 

entiendo que esa es la forma de indicar las funciones que se espera que implementemos ¿No?

Espero su respuesta,
Gracias

**Comment by martinvilu** [comment, marked as answer]
> Hola, para el punto 1, cuando dice "SOLDOUT" solo se refiere a imprimir en pantalla el Hola mundo, no? o también tiene que ver con alguna forma de codificar, o de hacer el ejercicio? 
> 

STDOUT es el printf


> En algunos puntos, aparecen por ejemplo:
> 
> float centigrados_a_fahrenheit(float centigrados); 
> 
> entiendo que esa es la forma de indicar las funciones que se espera que implementemos ¿No?

Si, no en algunos, en todos menos el primero.


> 
> Espero su respuesta,
> Gracias
> 
>

**Comment by AnGys09** [comment]
Ya que son varias consultas del TP1, estoy haciendo la corrección del TP1 y en el ejercicio 6, no logro que me de los resultados osea, las divisiones positivas  10/2= 5 resto 0, OK la división menos por mas  -4/2= 2 resto 0, OK, ahora las demás condiciones menos por menos  -4/-2 o 4/-2 me tira las direcciones de memoria y si corrijo una cosa después me salta error en resto, ya me estoy volviendo loca..... como puedo solucionarlo????

**Reply by martinvilu** [reply]
No es buena idea hacer una pregunta en un hilo "contestado" porque no llegan las notificaciones.

¿Como es eso que tira las direcciones de memoria? Queres compartir el enlace al repositorio asi lo veo?
